examples/presigned-getobject-request-date.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Minio Javascript Library for Amazon S3 Compatible Cloud Storage, (C) 2015 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ // Note: YOUR-ACCESSKEYID, YOUR-SECRETACCESSKEY, my-bucketname and my-objectname
+ // are dummy values, please replace them with original values.
+
+var Minio = require('minio')
+
+var s3Client = new Minio.Client({
+  endPoint: 's3.amazonaws.com',
+  accessKey: 'YOUR-ACCESSKEYID',
+  secretKey: 'YOUR-SECRETACCESSKEY',
+  useSSL: true // Default is true.
+})
+
+// Presigned get object URL for my-objectname at my-bucketname, it expires in 7 days by default.
+var requestDate = new Date('2017-01-01');
+var presignedUrl = s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, requestDate, function(e, presignedUrl) {
```

Comment:
I've tested the fix using this example script. However, it kept failing.
The reason turned out to be the missing optional parameter `respHeaders` in the call. So, `requestDate` value was read as `respHeaders` and `requestDate` got the default `now` value. 
We have to add the `respHeaders` in the call:
`s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, requestDate, ...`
=>
`s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000,`**`{},`**`requestDate, ...`

Reply:
Sorry for that.

Fixed in 02331c94739b0e563a35e0c35bb82d615c1d2ab7